src/state/contexts/CustomersContext.js

```diff
@@ -10,6 +10,7 @@ export const CustomersContext = createContext({});
 const CustomersProvider = ({ children }) => {
   const history = useHistory();
   const [custInfo, setCustInfo] = useState({});
+  // const [pets, setPets] = useState({});
```

Comment:
why are these couple lines commented out?

Reply:
This was part of a test I was conducting when mapping through pets wasn't working in RenderPetProfile. I tried moving pet state from the PetsContext I had created and including it in CustomersContext. It helped with some rendering, but not all. I eventually found that elevating that piece of state to load in the dashboard allowed it to render in the RenderPetProfile child component. I removed the above line from here.